docs/components/datetimepicker.md

```diff
@@ -36,26 +36,32 @@ Set the `v-model` to [Single date time picker](#single-date-time-picker-v-model)
 />
 ```
 
-<!-- ### Single time
+### Single date, custom width
+
+Same as above, except that the input field has a set pixel width of `300`.
 
 <div>
   <KDateTimePicker
-    @change="newVal => emitVal2 = newVal"
-    placeholder="Please select a time"
-    mode="time"
+    v-model="currentValue1"
+    @change="newVal => emitVal1 = newVal"
+    placeholder="Please select a date"
+    mode="date"
+    width="300"
     :range="false"
   />
-  <div class="mt-6">Emitted value: <pre class="json">{{ currentValue2 }}</pre></div>
+  <div class="mt-6">Emitted value: <pre class="json">{{ JSON.stringify(emitVal1) }}</pre></div>
 </div>
 
 ```html
 <KDateTimePicker
+  v-model="currentValue"
   @change="newVal => emitVal = newVal"
-  placeholder="Please select a time"
-  mode="time"
+  placeholder="Please select a date"
+  mode="date"
+  width="300"
   :range="false"
 />
-``` -->
+```
```

Comment:
TBH I don't know if we need a repetitive example just showing how to customize the width. Instead, maybe just customize the width on the existing example above and add a note?

Reply:
oh, duh. I don't know why I didn't think of that 🤦